exporter/awsemfexporter/grouped_metric.go

```diff
@@ -202,8 +222,8 @@ func translateUnit(metric pmetric.Metric, descriptor map[string]MetricDescriptor
 		unit = "Microseconds"
 	case "By":
 		unit = "Bytes"
-	case "Bi":
+	case "bit":
```

Comment:
We have this wrong the CloudWatch Agent as well. https://github.com/aws/amazon-cloudwatch-agent/blob/main/internal/cloudwatch/unit.go#L31

I can fix it.